main.py

```diff
@@ -1,7 +1,31 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Genre.objects.filter(name="Action").delete()
+
+    for first_name_to_set, last_name_to_set in actors:
```

Comment:
No previous was good. first_name, last_name is good here if you create an object)